rs/consensus/src/certification/certifier.rs

```diff
@@ -75,6 +81,8 @@ impl<Pool: CertificationPool> BouncerFactory<CertificationMessageId, Pool> for C
     // any new artifacts at that height. If it is above the CUP height and we do not
     // have a full certification at that height, we're interested in all artifacts.
     fn new_bouncer(&self, certification_pool: &Pool) -> Bouncer<CertificationMessageId> {
+        let _timer = self.metrics.start_timer();
```

Comment:
I'm not sure this abstractions make it easy to follow what timer is being started. I would prefer if we just use the metric field directly, as it makes it less confusing what timer you are starting.
```suggestion
        let _timer = self.metrics.update_duration.start_timer();
```